o/oas-kit/oas-kit.sh

```diff
@@ -0,0 +1,61 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: oas-kit
+# Version	: v5.0.0
+# Source repo	: https://github.com/Mermade/oas-kit
+# Tested on	: UBI 8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: BulkPackageSearch Automation {maintainer}
```

Comment:
this should be updated to you name + email?

Reply:
Updated, please review